plugins/unifi/unifi_clients_by_device

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/perl
+# -*- perl -*-
+
+=encoding utf8
+
+=head1 NAME
+
+unifi_clients_by_device - Munin plugin to display clients connected 
+for each ubiquity / unifi device.
+
+=head1 APPLICABLE SYSTEMS
+
+Currently is cabable of polling the unifi line (Ubiquity) from the unifi 
+API - requires direct access to the controllor API.
+
+=head1 CONFIGURATION
+
+This script uses the multigraph functionality to generate many graphs.
+
+For multigraph operation, the "internal" graphs will be one per device
+with seperate metrics for users and guests. The main graph shows total
+clients.
+
+If you are only interested in a single combined graph, you can set:
+
+  [unifi_clients_by_device]
+    env.multigraph no
```

Comment:
I'm of two minds about this - the additional graphs don't impact my system heavily - but my site configuration only features 8 devices, so it's not a ton of data, even extrapolated like this - my concern is, on a site with many, many unifi devices, there would be a *huge* amount of subgraphs. Of particular concern is the unifi_xfer_by_port plugin - if I had even a dozen 48 port switches it goes from a handful of graphs to something like 500+.  Perhaps it can be better named - subgraphs_enabled or something?

Reply:
You are right - the bigger environments would probably be happy about that configuration setting.
How about `enable_detail_graphs`? But maybe you can make up a better name.